mycore-base/src/main/resources/xsl/functions/classification.xsl

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0"?>
+<xsl:stylesheet version="3.0" xmlns:xsl="http://www.w3.org/1999/XSL/Transform"
+  xmlns:xlink="http://www.w3.org/1999/xlink" 
+  xmlns:fn="http://www.w3.org/2005/xpath-functions"
+  xmlns:xs="http://www.w3.org/2001/XMLSchema"
+  xmlns:mcrclass="http://www.mycore.de/xslt/classification"
+  exclude-result-prefixes="fn">
+  <xsl:function name="mcrclass:current-label" as="xs:string?">
+    <xsl:param name="class" as="element()" />
```

Comment:
should also work with a specific (root) category to work with complete classifications, too